src/main/resources/application.properties

```diff
@@ -3,4 +3,4 @@ db.url=jdbc:mysql://localhost:3306/tickets?serverTimezone=Europe/Kiev
 db.username=root
 db.password=root
 hibernate.show_sql=true
-hibernate.hbm2ddl.auto=create
+hibernate.hbm2ddl.auto=update
```

Comment:
let's not change this

Reply:
If i not change this my code starts more than 10 mins